code/modules/food&drinks/kitchen machinery/microwave.dm

```diff
@@ -251,6 +251,7 @@
 			if(F.cooked_type)
 				var/obj/item/weapon/reagent_containers/food/snacks/S = new F.cooked_type (get_turf(src))
 				F.initialize_cooked_food(S, efficiency)
+				feedback_add_details("food_made","[F.name]")
```

Comment:
Recommend you replace with type or some other identifier instead of name, name is often changed by admins or code